RealmSwift/Sync.swift

```diff
@@ -966,7 +966,9 @@ public class UserPublisher: Publisher {
     /// :nodoc:
     public func receive<S>(subscriber: S) where S: Subscriber, S.Failure == Never, Output == S.Input {
         let token = user.subscribe { _ in
-            _ = subscriber.receive(self.user)
+            DispatchQueue.main.async {
```

Comment:
This is not a valid thing to do, and it's producing (correct) sendability warnings.

Reply:
I'll revert this. Is the correct approach to have the end user bounce to the main thread instead when calling something which mutates the`User`?